.github/workflows/ci-eventbrite-consents.yml

```diff
@@ -0,0 +1,54 @@
+name: CI - Eventbrite consents sync lambda
+
+on:
+  pull_request:
+    paths:
+      - 'eventbrite-consents/**'
+      - '.github/workflows/ci-eventbrite-consents.yml'
+  push:
+    branches:
+      - main
+    paths:
+      - 'eventbrite-consents/**'
+      - '.github/workflows/ci-eventbrite-consents.yml'
+  workflow_dispatch:
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # required by aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read        
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Java 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'corretto'
+          cache: 'sbt'
+
+      - name: Test and build 
+        run: sbt clean compile test assembly
+        working-directory: eventbrite-consents
+
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+          aws-region: eu-west-1
+
+      - name: Upload to RiffRaff
+        uses: guardian/actions-riff-raff@v2
+        with:
+          projectName: eventbrite-consents-lambda
+          buildNumberOffset: 82
+          configPath: eventbrite-consents/riff-raff.yaml
+          contentDirectories: |
+            eventbrite-consents-lambda:
+              - eventbrite-consents/target/scala-2.12/main.jar
```

Comment:
I struggled in getting this to work sometimes, because the scala version was different from what I got locally, so I moved it to the root dir and renamed, if you see anything like this:
```yaml
- run: |
    sbt compile test assembly
    mv target/scala*/stupidly-named.jar ./good-name.jar
```
you'll know why